src/webview/handleOutboundEvents.js

```diff
@@ -214,11 +214,19 @@ const handleLongPress = (
   const { dispatch, showActionSheetWithOptions, backgroundData, narrow, startEditMessage } = props;
   if (target === 'header') {
     if (message.type === 'stream') {
+      const stream = streamNameOfStreamMessage(message);
+      const subscription = backgroundData.subscriptions.find(x => x.name === stream);
+
+      if (!subscription) {
```

Comment:
(same comment as above, about whether we expect this to happen, and `invariant`)

Reply:
using invariant here, the reason is same as https://github.com/zulip/zulip-mobile/pull/4635#discussion_r632862010